packages/entity-browser/src/components/FormField/fieldFactory.js

```diff
@@ -0,0 +1,68 @@
+  'ch.tocco.nice2.model.form.components.simple.DocumentField': null,
+  'ch.tocco.nice2.model.form.components.simple.DurationField': null,
+  'ch.tocco.nice2.model.form.components.simple.TimeField': null,
+  'ch.tocco.nice2.model.form.components.simple.PulldownDateField': null,
+  'ch.tocco.nice2.model.form.components.simple.UploadField': null,
+  'ch.tocco.nice2.model.form.components.simple.NamedUploadField': null,
+  'ch.tocco.nice2.model.form.components.simple.DmsDocumentField': null,
+  'ch.tocco.nice2.model.form.components.simple.DocumentFolderField': null,
+  'ch.tocco.nice2.model.form.components.simple.ImageField': null,
+  'ch.tocco.nice2.model.form.components.simple.ListTextArea': null,
+  'ch.tocco.nice2.model.form.components.simple.UuidField': null,
+  'ch.tocco.nice2.model.form.components.simple.PathField': null,
+  'ch.tocco.nice2.model.form.components.simple.StatusField': null,
+  'ch.tocco.nice2.model.form.components.simple.HtmlField': null,
+  'ch.tocco.nice2.model.form.components.simple.CodeField': null,
+  'ch.tocco.nice2.model.form.components.simple.LinkField': null,
+  'ch.tocco.nice2.model.form.components.simple.ConstantField': null,
+  'ch.tocco.nice2.model.form.components.simple.ListPanel': null,
+  'ch.tocco.nice2.model.form.components.simple.ReferencesListPanel': null,
+  'ch.tocco.nice2.model.form.components.simple.PasswordField': null,
+  'ch.tocco.nice2.model.form.components.simple.CreatePasswordField': null,
+  'ch.tocco.nice2.model.form.components.simple.RangeField': null,
+  'ch.tocco.nice2.model.form.components.simple.DisplayField': null,
+  'ch.tocco.nice2.model.form.components.simple.DisplayExpressionFielxpressionFieldFacade': null,
+  'ch.tocco.nice2.model.form.components.simple.MoneyAmountField': null,
+  'ch.tocco.nice2.model.form.components.simple.PercentField': null,
+  'ch.tocco.nice2.model.form.components.simple.DataAmountField': null,
+  'ch.tocco.nice2.model.form.components.simple.CustomDataField': null,
+  'ch.tocco.nice2.model.form.components.table.Table': null,
+  'ch.tocco.nice2.model.form.components.table.Column': null,
+  'ch.tocco.nice2.model.form.components.action.Action': null,
+  'ch.tocco.nice2.model.form.components.action.ActionSeparator': null,
+  'ch.tocco.nice2.model.form.components.Form': null,
+  'ch.tocco.nice2.model.form.components.Template': null,
+  'ch.tocco.nice2.model.form.components.layout.HorizontalBox': null,
+  'ch.tocco.nice2.model.form.components.layout.VerticalBox': null,
+  'ch.tocco.nice2.model.form.components.navigation.IteratorComponent': null,
+  'ch.tocco.nice2.model.form.components.simple.DescriptionField': null,
+  'ch.tocco.nice2.model.form.components.composite.LocationField': null,
+  'ch.tocco.nice2.model.form.components.simple.LoginField': null
+}
+
+export default (formField, modelField, props, events, utils) => {
+  const typeFactory = typeFactoryMap[formField.type]
+  if (!typeFactory) return <span/>
```

Comment:
shouldn't we log an error or at least a warning if the form field type (-> the key in the map) is not present at all in `typeFactoryMap`?